deeplab/package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "@tensorflow-models/deeplab",
+  "version": "0.1.0",
+  "description": "Model starter in TensorFlow.js",
+  "main": "dist/index.js",
+  "jsnext:main": "dist/deeplab.esm.js",
+  "module": "dist/deeplab.esm.js",
+  "unpkg": "dist/deeplab.min.js",
+  "jsdelivr": "dist/deeplab.min.js",
+  "types": "dist/index.d.ts",
+  "model": {
+    "name": "deeplab"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sdll/tfjs-model-starter"
```

We should change this:

```suggestion
    "url": "https://github.com/tensorflow/tfjs-models"
```